### PR TITLE
Frontend/about us page

### PR DESCRIPTION
**Pull request by PureDreamer**
This is the about us page.
It contains an explanation about the project and the course

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def create_tables(engine, psql_environment):
 from app.routers import (  # noqa: E402
     agenda, calendar, categories, celebrity, currency, dayview,
     email, event, four_o_four, invitation, profile, search,
-    weekview, telegram, whatsapp,
+    weekview, telegram, whatsapp, about_us,
```
Comment (yammesicka) [inline app/main.py:40]: Add in the correct place, please.

File: app/main.py [hunk 2]
```diff
@@ -59,6 +59,8 @@ def create_tables(engine, psql_environment):
     search.router,
     telegram.router,
     whatsapp.router,
+    about_us.router,
```
Comment (yammesicka) [inline app/main.py:62]: ^

File: app/static/about.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    align-items: center;
+    height: 100vh;
+    background-color: #F7F7F7;
+}
+
+.moving-words {
+    font-family: "Assistant", "Ariel", sans-serif;
+    text-transform: uppercase;
+    font-size: 6em;
+    letter-spacing: 4px;
```
Comment (yammesicka) [inline app/static/about.css:14]: use em

File: app/templates/calendar/layout.html [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    <!-- Fonts -->
+    <link href="https://fonts.googleapis.com/css2?family=Assistant:wght@200;300;400;500;600;700;800&display=swap"
+        rel="stylesheet">
+
+    <!-- CSS -->
+    <link href="{{ url_for('static', path='/grid_style.css') }}" rel="stylesheet" type="text/css">
+
+    <!-- Scripts -->
+    <script src="{{ url_for('static', path='/js/grid_scripts.js') }}"></script>
+    <script src="https://unpkg.com/ionicons@5.2.3/dist/ionicons.js"></script>
```
Comment (yammesicka) [inline app/templates/calendar/layout.html:18]: JS Should appear before the `</body>`

File: app/templates/profile.html [hunk 5]
```diff
@@ -1,338 +1,315 @@
 {% extends "base.html" %} {% include "event/partials/text_editor_partial_head.html" %} {% block content %}
 
-  <div class="container mt-4">
+<div class="container mt-4">
     <div class="row">
-      <!-- Left side -->
-      <div class="col-3">
-        <!-- Profile card -->
-        <div class="card card-profile mb-3">
+        <!-- Left side -->
```
Comment (yammesicka) [inline app/templates/profile.html:5]: Please return to the original state of 2 indentations

File: tests/utils.py [hunk 6]
```diff
@@ -0,0 +1,13 @@
+from sqlalchemy.orm import Session
```
Comment (yammesicka) [inline tests/utils.py:1]: Why have you added an empty `conftest.py` file?

**Review by yammesicka — COMMENTED** [review]
It's a good start, please look at the insights and have a frontend team member look at this commit :)

File: tests/utils.py (hunk 6 above)
Comment (PureDreamer) [reply tests/utils.py:1]: this is not my work.

Comment (PureDreamer) [reply tests/utils.py:1]: I will delete it although.

File: app/templates/calendar/layout.html (hunk 4 above)
Comment (PureDreamer) [reply app/templates/calendar/layout.html:18]: also not my work.

File: app/static/about.css (hunk 3 above)
Comment (PureDreamer) [reply app/static/about.css:14]: changed!

File: app/main.py (hunk 2 above)
Comment (PureDreamer) [reply app/main.py:62]: changed!

File: app/main.py (hunk 1 above)
Comment (PureDreamer) [reply app/main.py:40]: changed!

File: app/static/about.css [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    background-color: #F7F7F7;
+}
+
+.moving-words {
+    font-family: "Assistant", "Ariel", sans-serif;
+    text-transform: uppercase;
+    font-size: 6em;
```
Comment (aviadamar) [inline app/static/about.css:13]: please use rem units and not em.

File: app/static/about.css [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+    margin: 0;
+    padding: 0;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    background-color: #F7F7F7;
+}
+
+.moving-words {
+    font-family: "Assistant", "Ariel", sans-serif;
```
Comment (aviadamar) [inline app/static/about.css:11]: this is already set the the global.css, please connect it to your pages.

File: app/static/about.css [hunk 9]
```diff
@@ -0,0 +1,56 @@
+body {
+    margin: 0;
```
Comment (aviadamar) [inline app/static/about.css:2]: this is already set the the global.css, please connect it to your pages. 
(check for all those general settings)

File: app/static/about.css [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+    margin-right: auto;
+    width: 100%;
+}
+
+.abouttext {
+    font-size: 1.2em;
+    text-align: justify;
+}
+
+a {
```
Comment (aviadamar) [inline app/static/about.css:49]: this is already set the the global.css, please connect it to your pages.

File: app/templates/calendar/layout.html [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+                    </div>
+                    <div>
+                        <ion-icon name="settings-outline"></ion-icon>
+                    </div>
+                    <div>
+                        <ion-icon name="calendar-outline"></ion-icon>
+                    </div>
+                    <div>
+                        <ion-icon name="cube-outline"></ion-icon>
+                    </div>
```
Comment (aviadamar) [inline app/templates/calendar/layout.html:43]: Please enter a link to the about us page here after that <div> 
with the icon of:
```<ion-icon name="information-circle-outline"></ion-icon>```

put it in a link <a> tag - no need to set any type of class for it.

File: app/templates/about_us.html [hunk 12]
```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
```
Comment (aviadamar) [inline app/templates/about_us.html:1]: there is no need to create a full new HTML page for getting the navigation bar. 
Please inherent the navigation:
I'd recommand to copy the "calendar_base.html" file and put your content inside: 
```
    {% block content %}
    {% endblock content %}
   ```

File: app/templates/about_us.html [hunk 13]
```diff
@@ -0,0 +1,101 @@
...
+            <ul>
+                <li> Want to monitor your expenses? We got it.</li>
+                <li>Want to organize a zoom meeting while still under quarantine? We got it.</li>
+            </ul>
+            <p class="abouttext">Even better - you got it! Develop your own feature and add it to the calendar. This calendar is open source developed mostly in Python as the finale to an extraordinary course. For the calendar development we selected features, aiming at required
+                and unique. Then each of us began the journey to implement it in the calendar. We worked collaboratively to put it together.
+                <br> if you like what you just read, please <a href="sign_in">sign in.</a>
+            </p>
+        </div>
+        <br><br><br><br>
```
Comment (aviadamar) [inline app/templates/about_us.html:78]: dont create white spaces as such. 
U can create <sections> tag and give it a:
```
margin-top: ..;
```

File: app/templates/about_us.html [hunk 14]
```diff
@@ -0,0 +1,101 @@
...
+        <blockquote class="blockquote text-center">
+            <p class="mb-0"><i>“If you want to go fast, go alone. If you want to go far, go together.”</i></p><br>
+            <footer class="blockquote-footer"><cite title="Source Title">African proverb</cite></footer>
+        </blockquote>
+        <br><br><br><br>
```
Comment (aviadamar) [inline app/templates/about_us.html:83]: dont create white spaces as such. 
U can create <sections> tag and give it a:
```
margin-top: ..;
```

File: app/templates/about_us.html [hunk 15]
```diff
@@ -0,0 +1,101 @@
...
+                        <a class="nav-link" href="/celebrity">Celebrities Born Today</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="{{ url_for('salary_home') }}">Salary</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <br>
```
Comment (aviadamar) [inline app/templates/about_us.html:54]: use margins and not ```<bt>``` for white space.

File: app/templates/about_us.html [hunk 16]
```diff
@@ -0,0 +1,101 @@
...
+    <div class="container">
+        <h1 class="moving-words">About The Project</h1>
+        <!--  credit for the animation goes to FrankieDoodie -->
+        <div>
+            <h3>Do you have a dream? How will you make it come true?</h3>
+            <p class="abouttext">
```
Comment (aviadamar) [inline app/templates/about_us.html:60]: Id change the class name to "about-text" .. but depends on you.

File: app/templates/about_us.html [hunk 17]
```diff
@@ -0,0 +1,101 @@
...
+                a course that has helped us set foot in the world of programming. The course took us from complete newbies to decent programmers. The course curriculum can be found in the attached infographic.</p><br> <br><br>
+            <p class="abouttext">The materials of the first half of the course can be found on GitHub, you can take a look right <a href="https://github.com/PythonFreeCourse/Notebooks">here.</a></p>
+        </div>
+        <div>
+            <img class="infographic" src="{{ url_for('media', path='free-python-course-4k.png') }}" alt="Course infographic">
+        </div>
+
+        <div>
+            <h2> We thank you for letting us share our story and hope you will enjoy our project.</h2>
+            <p>If you enjoy it please consider giving us a star in our <a href="https://github.com/PythonFreeCourse/calendar">GitHub repository.</a></p>
```
Comment (aviadamar) [inline app/templates/about_us.html:96]: add ```title=..``` property to to your ```<a>``` element.

File: app/templates/about_us.html (hunk 16 above)
Comment (PureDreamer) [reply app/templates/about_us.html:60]: changed

File: app/templates/about_us.html (hunk 17 above)
Comment (PureDreamer) [reply app/templates/about_us.html:96]: changed

File: app/templates/about_us.html (hunk 15 above)
Comment (PureDreamer) [reply app/templates/about_us.html:54]: changed!

File: app/templates/about_us.html (hunk 14 above)
Comment (PureDreamer) [reply app/templates/about_us.html:83]: changed!

File: app/templates/about_us.html (hunk 13 above)
Comment (PureDreamer) [reply app/templates/about_us.html:78]: changed!

File: app/static/about.css (hunk 10 above)
Comment (PureDreamer) [reply app/static/about.css:49]: added.

File: app/static/about.css (hunk 9 above)
Comment (PureDreamer) [reply app/static/about.css:2]: added!

File: app/static/about.css (hunk 8 above)
Comment (PureDreamer) [reply app/static/about.css:11]: changed!

File: app/static/about.css (hunk 7 above)
Comment (PureDreamer) [reply app/static/about.css:13]: changed!

File: app/templates/calendar/layout.html (hunk 11 above)
Comment (PureDreamer) [reply app/templates/calendar/layout.html:43]: added it to the about page!

File: app/templates/about_us.html (hunk 12 above)
Comment (PureDreamer) [reply app/templates/about_us.html:1]: I didnt use this instead I added 
`{% extends "base.html" %}`

Comment (PureDreamer) [reply app/templates/about_us.html:1]: and Ive added the 
`    {% block content %}
    {% endblock content %}`

File: tests/test_about.py [hunk 18]
```diff
@@ -0,0 +1,4 @@
+def test_about_page(client):
+    response = client.get("/about")
+    assert response.ok
+    assert b"Yam" in response.content
```
Comment (yammesicka) [inline tests/test_about.py:4]: XD

**Review by yammesicka — APPROVED** [review]

File: app/main.py [hunk 19]
```diff
@@ -88,6 +89,8 @@ async def swagger_ui_redirect():
     telegram.router,
     user.router,
     whatsapp.router,
+
```
Comment (ivarshav) [inline app/main.py:92]: Redundent?

Comment (PureDreamer) [reply app/main.py:92]: fixed

**Review by yammesicka — APPROVED** [review]

**Pull request merged**